inspirehep/modules/workflows/tasks/submission.py

```diff
@@ -345,8 +345,7 @@ def prepare_keywords(obj, eng):
         # TODO: differentiate between curated and gueesed keywords
         keywords.append(
             {
-                'classification_scheme': '',
-                'keyword': keyword['label'],
+                'value': keyword['label'],
```

Comment:
Yes, I think so. The schema says: `Leave empty for free form keywords`

But I guess the `magpie` keywords should be using the INSPIRE schema.